dashboard/app/models/level.rb

```diff
@@ -35,6 +36,7 @@ class Level < ActiveRecord::Base
   has_many :hint_view_requests
 
   before_validation :strip_name
+  before_save :log_changes
```

Comment:
With this also be called when seeding levels?  Seems like this may cause conflicts between different environments.